content/help.md

```diff
@@ -1,38 +1,23 @@
 ---
-title: Help
-url: "help"
+title: Get Involved
+url: "community"
 ---
 
+## Hang out
 
-## Get help
+Get live support and talk with other lovers of Odin on the [Odin Discord](https://discord.gg/sVBPHEv).
 
-### [Odin Discord](https://discord.gg/sVBPHEv)
+## Report issues
 
-Get live support and talk with other lovers of Odin on the Odin Discord.
+If you find a bug, mistake, or an inconsistency in the Odin code or documentation, please let us know by [opening an issue](https://github.com/odin-lang/odin/issues/new).
 
-
-### [Frequently Asked Questions (FAQ)](/docs/faq)
-
-Answers to common questions about Odin.
-
-
-### [Source Code](https://github.com/odin-lang/Odin)
-
-Check out the Odin source code.
```

Comment:
Having the FAQ is helpful. The FAQ can contain many ways on how to help too, it's not just concerning the language itself.

Reply:
This depends on if the primary purpose of the page is for help or not. I think Community makes a lot more sense. 

If they want to self-serve, they'll go to docs. If they want help, they'll want to reach out to the community.